src/main/java/com/iexec/worker/tee/scone/SconeTeeService.java

```diff
@@ -0,0 +1,93 @@
+package com.iexec.worker.tee.scone;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Optional;
+
+import com.iexec.common.chain.ContributionAuthorization;
+import com.iexec.common.security.Signature;
+import com.iexec.common.sms.scone.SconeSecureSessionResponse.SconeSecureSession;
+import com.iexec.common.utils.BytesUtils;
+import com.iexec.common.utils.HashUtils;
+import com.iexec.common.utils.SignatureUtils;
+import com.iexec.worker.config.PublicConfigurationService;
+import com.iexec.worker.config.WorkerConfigurationService;
+import com.iexec.worker.sms.SmsService;
+import com.iexec.worker.utils.FileHelper;
+
+import org.springframework.stereotype.Service;
+
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+@Service
+public class SconeTeeService {
+
+    private static final String FSPF_FILENAME = "volume.fspf";
```

Comment:
What is it used for? Could document it a bit please?

Reply:
Sure, I'll add a short description